project_generation/content/templates/event/service/service.go.tmpl

```diff
@@ -35,6 +37,8 @@ func Run(ctx context.Context, serviceList *ExternalServiceList, buildTime, gitCo
 
 	// Get HTTP Server with collectionID checkHeader middleware
 	r := mux.NewRouter()
+	otelHandler := otelhttp.NewHandler(router, "/")
```

Comment:
When I generate a new event driven project, I get an error "undefined: router"

Reply:
I've fixed this and added in the instrumentation here now, but am having running the project including in develop